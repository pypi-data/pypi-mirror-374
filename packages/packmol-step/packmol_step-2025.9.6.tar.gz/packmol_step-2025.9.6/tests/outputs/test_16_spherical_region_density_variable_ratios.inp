tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside sphere 0.0 0.0 0.0 16.3439
   number 75
end structure
structure input_2.pdb
   inside sphere 0.0 0.0 0.0 16.3439
   number 225
end structure
