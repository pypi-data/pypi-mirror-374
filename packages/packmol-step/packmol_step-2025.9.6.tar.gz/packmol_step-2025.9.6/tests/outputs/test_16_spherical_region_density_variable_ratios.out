    Will create a spherical region containing the following molecules:

        +-------------+-------------+---------+
        |  Component  | Structure   |   Ratio |
        |-------------+-------------+---------|
        |    fluid    | c1ccccc1    |       1 |
        |    fluid    | O           |       3 |
        +-------------+-------------+---------+

    The dimensions of the region will be calculated from the density 0.9 g/ml.
    The number of molecules of the fluid will be obtained by rounding 300
    molecules to give the requested ratios of species.

Created a spherical region with a diameter of 32.69 Ã… with 300 fluid molecules

    +-------------+-------------+---------------+----------+------------+
    |  Component  | Structure   |   Requested % |   Number |   Actual % |
    |-------------+-------------+---------------+----------+------------|
    |    fluid    | c1ccccc1    |            25 |       75 |         25 |
    |    fluid    | O           |            75 |      225 |         75 |
    +-------------+-------------+---------------+----------+------------+

There are a total of 1575 atoms in the cell giving a density of 0.9 g/ml.